compiler/test/fail_compilation/fail13577.d

```diff
@@ -0,0 +1,28 @@
+// https://issues.dlang.org/show_bug.cgi?id=13577
+
+/*
+TEST_OUTPUT:
+---
+fail_compilation/fail13577.d(27): Error: cannot implicilty convert range element of type `int[]` to parameter `x` of type `immutable(int[])`
```

Comment:
Parameter?

Reply:
Well, technically you can have any number of such variables, provided that front returns a tuple. With that in mind, I guess I thought of them as parameters to the tuple. I wanted a more generic term, in case users will have more than 2 variables and the mismatch occurs to one of them.

Reply:
It's kind of tricky to get a wording that specifically tells what's happening. The idea would be that one of the elements of the result tuple of front does match match the type of a given variable in the foreach list of variables. However, I find this too complicated to be a user facing message.